### PR TITLE
Feature/setup user

### PR DESCRIPTION
**Pull request by thuynguyen650**

### CODE REVIEW

File: src/LibraryManagement.Domain/Entities/WaitingList.cs
Comment (Kira-UIT) [inline src/LibraryManagement.Domain/Entities/WaitingList.cs:1]: Rename from "WaitingList.cs" to "UserBook.cs"

File: src/LibraryManagement.Infrastructure/Persistence/EntitiesConfiguration/UserConfiguration.cs
Comment (Kira-UIT) [inline src/LibraryManagement.Infrastructure/Persistence/EntitiesConfiguration/UserConfiguration.cs:1]: Change "WaitingLists" to "Books" in User entity

File: src/LibraryManagement.Infrastructure/Persistence/EntitiesConfiguration/WaitingListConfiguration.cs
Comment (Kira-UIT) [inline src/LibraryManagement.Infrastructure/Persistence/EntitiesConfiguration/WaitingListConfiguration.cs:1]: Change "WaitingList" to "UserBook"

**Review by Kira-UIT — CHANGES_REQUESTED** [review]
Need to change something

File: src/LibraryManagement.Client/Controllers/UsersController.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{
+    private ApplicationDbContext _context;
+
+    public UsersController(ApplicationDbContext context)
+    {
+        _context = context;
+    }
+
+    [HttpGet]
+    public async Task<List<User>> GetUsers()
```
[the remark this answers is not in the document]
Comment (thuynguyen650) [reply src/LibraryManagement.Client/Controllers/UsersController.cs]: Updated

File: src/LibraryManagement.Domain/Entities/WaitingList.cs
Comment (thuynguyen650) [reply src/LibraryManagement.Domain/Entities/WaitingList.cs:1]: Removed WaitingList

File: src/LibraryManagement.Infrastructure/Persistence/EntitiesConfiguration/WaitingListConfiguration.cs
Comment (thuynguyen650) [reply src/LibraryManagement.Infrastructure/Persistence/EntitiesConfiguration/WaitingListConfiguration.cs:1]: Removed WaitingList

**Review by Kira-UIT — APPROVED** [review]
Good job

**Pull request merged**